suites/nautilus/rgw/tier-1_containerized_rgw_ssl_regression-suite.yaml

```diff
@@ -0,0 +1,107 @@
+tests:
+   - test:
+       name: install ceph pre-requisites
+       module: install_prereq.py
+       abort-on-fail: true
+   - test:
+      name: containerized ceph ansible
+      polarion-id: CEPH-83571503
+      module: test_ansible.py
+      config:
+        ansi_config:
+            ceph_test: True
+            ceph_origin: distro
+            ceph_stable_release: nautilus
+            ceph_repository: rhcs
+            osd_scenario: lvm
+            osd_auto_discovery: False
+            journal_size: 1024
+            ceph_stable: True
+            ceph_stable_rh_storage: True
+            ceph_docker_image: rhceph/rhceph-3-rhel7
+            ceph_docker_image_tag: latest
+            containerized_deployment: true
+            ceph_docker_registry: registry.access.redhat.com
+            copy_admin_key: true
+            dashboard_enabled: False
+            ceph_conf_overrides:
+                global:
+                  osd_pool_default_pg_num: 64
+                  osd_default_pool_size: 2
+                  osd_pool_default_pgp_num: 64
+                  mon_max_pg_per_osd: 1024
+                mon:
+                  mon_allow_pool_delete: true
```

Comment:
Do we need these overrides? If the test suite indicates, please add the reason for inclusion in the docString.